filesystem/src/main/java/org/frankframework/filesystem/TypeFilter.java

```diff
@@ -0,0 +1,22 @@
+/*
+   Copyright 2024 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package org.frankframework.filesystem;
+
+public enum TypeFilter {
+	FILES_ONLY,
+	FOLDERS_ONLY,
+	FILES_AND_FOLDERS
+}
```

Comment:
If you add to this enum attributes `includeFiles` and `includeFolders` then a lot of code can perhaps be simplified.
Instead of checking 
`if (typeFilter == FILES_ONLY || typeFilter == FILES_AND_FOLDERS) { /* add files */ }`

You can check
` if (typeFilter.includeFiles) { /* add files */ }`

Reply:
Yeah, sounds like a good idea. Thanks!